File: Client/src/client/systems/marker.js
/// <reference types="@altv/types-client" />
/// <reference types="@altv/types-natives" />

import alt from 'alt-client';
import native from 'natives';

function createGarageMarker(storagePositionList) 
{
    const markerOptions = {
        color: new alt.RGBA(231, 114, 0, 255),
        type: 1
    };

    storagePositionList.forEach(function(markerPosition) 
    {
        const marker = new alt.Utils.Marker(markerPosition, markerOptions);
        marker.scale = new alt.Vector3(3.0);
    });
}

alt.onServer('Client:Marker:Garage', (storagePositionList) => {
    createGarageMarker(storagePositionList);
});

File: Los Angeles Life Client/Los Angeles Life Client/System/Marker.cs
using AltV.Net.Client;
using AltV.Net.Data;

namespace Los_Angeles_Life_Client.System;

public abstract class Marker : Client
{
    public static void Load()
    {
        /*
        Alt.OnServer("Client:Marker:Garage", (List<Position> positions) =>
        {
            CreateMarker(positions);
        });
        */
    }

    private static void CreateMarker(List<Position> positions)
    {
        foreach(Position position in positions)
        {
            Alt.Log("Marker ertellt: " + position.ToString());
        }
    }
}